firebase-functions/src/androidTest/backend/functions/package.json

```diff
@@ -2,14 +2,11 @@
   "name": "functions",
   "description": "Cloud Functions for Firebase",
   "dependencies": {
-    "firebase-admin": "~4.2.1",
-    "firebase-functions": "^0.5.7"
-  },
-  "devDependencies": {
-    "@google-cloud/functions-emulator": "1.0.0-beta.4"
```

Comment:
We don't need the emulator anymore?

Reply:
This npm module has been deprecated: https://www.npmjs.com/package/@google-cloud/functions-emulator. One of the recommended alternative is to start emulator with `firebase-tools`, which is the same tool used to deploy functions. So I think it's no longer needed here.